units/breakpoint-pre-mount.service.in

```diff
@@ -0,0 +1,38 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Breakpoint before mounting the root filesystem on /sysroot
+Documentation=man:systemd-debug-generator(8)
+AssertPathExists=/etc/initrd-release
+DefaultDependencies=no
+Conflicts=shutdown.target emergency.target
+After=sysinit.target systemd-vconsole-setup.service
+Before=initrd-root-fs.target sysroot.mount systemd-fsck-root.service
+
+[Service]
+Environment=TERM=linux
+Environment=PS1="pre-mount:$PWD# "
+Type=oneshot
+# Execute shell with "-" prefix to not consider the unit failed if it exits with
+# a non-zero value
+ExecStartPre=-plymouth --wait quit
+ExecStart=-{{SUSHELL}}
+StandardInput=tty-force
+StandardOutput=inherit
+StandardError=inherit
+TasksMax=infinity
```

Comment:
Where does this come from?

Reply:
uh, i am not convinced this is a good idea. i don#t see why xfsrepair would require a ridiculous amount of tasks like this. this seems really weird...

Reply:
Ok, this came in the context of the emergency shell, but this PR is about adding debug breakpoints, so let's remove it.